docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: "3.8"
+
+services:
+  acezero:
+    restart: always
+    container_name: acezero
+    build:
+      context: .
+      dockerfile: Dockerfile
+    tty: true
+    stdin_open: true
+    ports:
+      - "7771:7007"
+    volumes:
+      - ./images:/workspace/images
+    deploy:
+      resources:
+        reservations:
+          devices:
+            - driver: 'nvidia'
+              count: all
+              capabilities: [gpu]
```

Comment:
I usually use `docker run --gpus='all,"capabilities=compute,utility,graphics"' ...`, now that might not be the most up to date way, but I **think** `capabilities=all` would be nice to have, see also here: https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/docker-specialized.html#driver-capabilities

Reply:
I think the docker-compose docs recommend using  capabilities: [gpu] for docker-compose files 

https://docs.docker.com/compose/how-tos/gpu-support/